### DISCUSSION TITLE
How about support for this lil guy NodeMCU ESP8266 Development Board with 0.96 Inch OLED Display

### DISCUSSION THREAD

**Discussion by DSCustoms** [dj1ch/minigotchi#132] [Ideas]
Not to add more work, I'm not sure exactly what would need to be changed, I've tried a few things and the minigotchi is running but I can't get this display to work. 

https://www.amazon.com/dp/B0BVM3H46B?psc=1

It's a great little board with the oled already onboard, but it needs this u8g2 library and commands.

https://github.com/olikraus/u8g2/wiki

https://kbin.social/m/Arduino/t/779187/Default-setup-and-screen-display-code-for-Ideaspark-ESP8266-096OLED

Anyone want to tackle this or give me a little idea what to start to change?

**Comment by dj1ch** [comment]
Hello @DSCustoms, I'm the main developer of this project and I'd be more than happy to support this display for the Minigotchi. As of right now, we do have SSD1306 support if you want to try messing around with that, but it appears that this special library is probably required to get this running. I'll create an issue with this and add this to my to-do list.

**Reply by DSCustoms** [reply]
Oh I know, I didn’t want to just throw this comment on reddit. I’ve built a few already. The weird thing is that this is an ssd1306 display, they just force this library to control it. I tried adding the library and including it, tried bypassing it, I just don’t have the specific experience to figure out what I missed. I’ve been testing different board configs to make it as accessible as possible.

**Comment by dj1ch** [comment]
Looks like we got it working if you want to try this on your ESP8266. You'll need to compile from the development branch
